src/main/java/de/zalando/tip/zalenium/proxy/DockerSeleniumStarterRemoteProxy.java

```diff
@@ -132,50 +147,50 @@ public float getResourceUsageInPercent() {
     /*
      * Reading configuration values from the environment variables, if a value was not provided it falls back to defaults.
      */
-    private static void readConfigurationFromFile() {
+    private static void readConfigurationFromEnvVariables() {
 
-        if (System.getenv("ZALENIUM_CHROME_CONTAINERS") != null) {
+        if (environment.getEnvVariable(ZALENIUM_CHROME_CONTAINERS) != null) {
             try {
-                int chromeContainers = Integer.parseInt(System.getenv("ZALENIUM_CHROME_CONTAINERS"));
+                int chromeContainers = Integer.parseInt(environment.getEnvVariable(ZALENIUM_CHROME_CONTAINERS));
                 setChromeContainersOnStartup(chromeContainers);
             } catch (Exception e) {
-                LOGGER.log(Level.WARNING, "[DS] ZALENIUM_CHROME_CONTAINERS {} is not a valid integer",
-                        System.getenv("ZALENIUM_CHROME_CONTAINERS"));
+                LOGGER.log(Level.WARNING, "[DS] Env. variable " + ZALENIUM_CHROME_CONTAINERS +
+                        " is not a valid integer", e);
                 setChromeContainersOnStartup(DEFAULT_AMOUNT_CHROME_CONTAINERS);
             }
         } else {
-            LOGGER.log(Level.FINE, "[DS] ZALENIUM_CHROME_CONTAINERS value is not set, falling back to default {}.",
-                    DEFAULT_AMOUNT_CHROME_CONTAINERS);
+            LOGGER.log(Level.FINE, "[DS] Env. variable " +  ZALENIUM_CHROME_CONTAINERS + " value is not set, " +
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal " value is not set, " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)